tests/integrations/opentelemetry/test_propagator.py

```diff
@@ -139,11 +140,47 @@ def test_inject_continue_trace(sentry_init):
         "HTTP_BAGGAGE": baggage,
     }
 
+    expected_baggage = baggage + ",sentry-sample_rand=0.002849"
+
+    with patch(
+        "sentry_sdk.tracing_utils.Random.uniform",
+        return_value=decimal.Decimal("0.002849"),
```

Comment:
`Random.uniform` returns a `float`, not a `Decimal`, so the mock return value should also be a `float`.

```suggestion
        return_value=0.002849,
```